src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styles from "./RequestTable.module.css";
+import classNames from "classnames";
+
+export const RequestRow = ({ onSeek, onClick, currentTime, prepareRow, row, selectedRequest }) => {
+  let currentRow = false;
+  // Did we just pass the row boundary that contains the current time?
+  // If so, let's render this row as the "current" row and all rows
+  // after it as future rows.
+  // if (
+  //   selectedRequest?.id === row.original.id ||
+  //   (!selectedRequest && renderingRequestsInThePast && currentTime <= row.original.point.time)
+  // ) {
+  //   renderingRequestsInThePast = false;
+  //   currentRow = true;
+  // }
```

Comment:
We'll need to refactor this to work in the new world... I think we could probably pass in a currentRowIndex which will be stable and an easy check

Reply:
I think the struggle with currentRowIndex is when things get filtered.